src/edu/mills/cs180a/RomanNumeralTest.java

```diff
@@ -27,7 +27,7 @@ public void equals_False_X10() {
   }
 
   @Test
-  public void reflective_true_XX_X10() {
+  public void symmetric_true_XX_X10() {
```

Comment:
Sorry for not catching this in the first review, but the test name is not in the right format. It should start with the name of the method under test, etc.  It should start `equals_True`. To tell you the truth, I'm not sure what the third part should be.

Reply:
I used X10XX for the third part.